safe/definitions/concepts.py

```diff
@@ -469,6 +469,33 @@
             }
         ],
     },
+    'thresholds': {
+        'key': 'thresholds',
+        'description': tr(
+            'A range that defined with minimum and maximum value. In InaSAFE '
+            'we exclude the minimum value but include the minimum value. In '
+            'mathematical expression: minimum value < x <= maximum value. It '
+            'is used for doing classification for continuous data.'),
```

Comment:
+            'A range defined with minimum and maximum values. In InaSAFE '
 +            'we exclude the minimum value but include the maximum value. '
 +            'Expressed mathematically: minimum value < x <= maximum value. Ranges '
 +            'are used for performing classification of continuous data.'),